utils/domain_string_to_felt.ts

```diff
@@ -0,0 +1,25 @@
+export default function domainStringToFelt(domain: string) {
```

Comment:
shouldn't you use our js sdk for this?

Reply:
We can't use it atm because I'll make a special PR for that to make sure it works well